k8s/client.go

```diff
@@ -1084,6 +1085,22 @@ func (c *Client) GetHelmState(ctx context.Context, namespace string, secretName
 	}, nil
 }
 
+// GetHelmValues is the function for cilium cli sysdump to collect the helm values from the release directly
+func (c *Client) GetHelmValues(_ context.Context, releaseName string) (string, error) {
+	client := action.NewGetValues(c.HelmActionConfig)
+	vals, err := client.Run(releaseName)
+	if err != nil {
+		return "", fmt.Errorf("unable to retrieve the helm value from the release %s: %w", releaseName, err)
+	}
+
+	valuesBuf := new(bytes.Buffer)
+	if err = output.EncodeYAML(valuesBuf, vals); err != nil {
+		return "", fmt.Errorf("unable to parse from helm values from release %s: %w", releaseName, err)
```

Comment:
This message is better as "unable to parse helm values from release %s: %w"

Reply:
will change in my next commit